source/common/grpc/google_async_client_impl.h

```diff
@@ -86,7 +86,9 @@ class GoogleAsyncClientThreadLocal : public ThreadLocal::ThreadLocalObject,
   void completionThread();
 
   // Instantiate this first to ensure grpc_init() is called.
+#ifdef ENVOY_GOOGLE_GRPC
```

Comment:
This whole file is already guarded with `ENVOY_GOOGLE_GRPC`. No need explicit guard here.

Reply:
Yep, seems redundant here.

Reply:
Removed the ifdef. Replicated the comment from source/extensions/transport_sockets/alts/config.cc